classloader-leak-prevention/classloader-leak-prevention-core/src/test/java/se/jiderhamn/classloader/leak/prevention/cleanup/ThreadLocalWithNestedRefValueCleanUpTest.java

```diff
@@ -0,0 +1,22 @@
+package se.jiderhamn.classloader.leak.prevention.cleanup;
+
+import java.lang.ref.SoftReference;
+
+/**
+ * Another variant test case for {@link ThreadLocalCleanUp} using chained {@link SoftReference}s to the value,
```

Comment:
I would have liked a reference to `ThreadLocalWithRefValueCleanUpTest` in the JavaDoc

Reply:
Ok no pb

Reply:
Done